zebra/subdir.am

```diff
@@ -104,7 +105,7 @@ zebra/debug_clippy.c: $(CLIPPY_DEPS)
 zebra/debug.$(OBJEXT): zebra/debug_clippy.c
 
 zebra/zebra_mlag_clippy.c: $(CLIPPY_DEPS)
-zebra/zebra_mlag.$(OBJEXT): zebra/zebra_mlag_clippy.c
+zebra/zebra_mlag.$(OBJEXT): zebra/zebra_mlag_vty_clippy.c
```

Comment:
You need to change the left side on these 2 rules too, that's why you're getting the clippy build error.

Reply:
Fixed